sources/packages/forms/src/form-definitions/educationprogramoffering.json

```diff
@@ -1,5759 +1,5763 @@
-                  "prefix": "",
-                  "customClass": "",
-                  "suffix": "",
-                  "multiple": false,
-                  "defaultValue": null,
-                  "protected": false,
-                  "unique": false,
-                  "persistent": false,
-                  "hidden": false,
-                  "clearOnHide": false,
-                  "refreshOn": "",
-                  "redrawOn": "",
-                  "modalEdit": false,
-                  "dataGridLabel": false,
-                  "labelPosition": "top",
-                  "description": "",
-                  "errorLabel": "",
-                  "tooltip": "",
-                  "tabindex": "",
-                  "disabled": false,
-                  "autofocus": false,
-                  "dbIndex": false,
-                  "customDefaultValue": "",
-                  "calculateValue": "",
-                  "calculateServer": false,
-                  "widget": null,
-                  "attributes": {},
-                  "validateOn": "change",
-                  "validate": {
-                    "required": false,
-                    "custom": "",
-                    "customPrivate": false,
-                    "strictDateValidation": false,
-                    "multiple": false,
-                    "unique": false
-                  },
-                  "conditional": {
-                    "show": null,
-                    "when": null,
-                    "eq": ""
-                  },
-                  "overlay": {
-                    "style": "",
-                    "left": "",
-                    "top": "",
-                    "width": "",
-                    "height": ""
-                  },
-                  "allowCalculateOverride": false,
-                  "encrypted": false,
-                  "showCharCount": false,
-                  "showWordCount": false,
-                  "properties": {},
-                  "allowMultipleMasks": false,
-                  "addons": [],
-                  "tree": false,
-                  "lazyLoad": false,
-                  "title": "Panel",
-                  "theme": "default",
-                  "breadcrumb": "default",
-                  "id": "ejk2389"
-                },
-                {
-                  "label": "Invalid study dates popup",
-                  "labelWidth": "",
-                  "labelMargin": "",
-                  "tag": "p",
-                  "className": "alert alert-warning fa fa-exclamation-triangle w-100",
-                  "attrs": [
-                    {
-                      "attr": "",
-                      "value": ""
-                    }
-                  ],
-                  "content": "<strong class=\"ml-2\">This study period is ineligible for StudentAid BC funding</strong>\n<br>\n<span>Your dates must be between {{data.studyPeriodMinDays}} to {{data.studyPeriodMaxDays}} days.</span>",

```

Comment:
Question to business and other devs.

This warning banner currently covers 2 validation and it's respective message. 

1. Minimum study period days  must be (84 for FT and 42 for PT)
2. Maximum study period days must be 365.

As per the current ticket we are updating condition 1. 

1. Minimum funded study weeks  must be (12 for FT and 6 for PT)
2. Maximum study period days must be 365.

Considering that the validation 1 is replaced by minimum study weeks. The warning banner message is now updated. 

But the updated warning banner message is completely ignoring the maximum 365 days validation context in it. 

Hence either we need to update the warning banner message to have a common context or have a separate banner for condition 2.